### PR TITLE
Work_addr_arg_starter

### PR DESCRIPTION
**Pull request by steffenkux**
* added the URL-command messageTo to call a messageBox directly to the given receiver
* added some changing graphics for the login screen
* removed button to call the old version (beta2)

image

### CODE REVIEW

File: packages/messenger-web/src/Dm3Widget.tsx [hunk 1]
```diff
@@ -1,17 +1,56 @@
...
 import { DM3, DM3Configuration } from '@dm3-org/dm3-messenger-widget';
 
-// Define the Dm3Widget component
+/**
+ * Helper function to generate a weekly-based pseudo-random image path.
+ * It uses the current week of the year to ensure the image remains consistent throughout the week,
+ * but changes weekly.
+ * @returns The path to the selected image.
+ */
+const getWeeklyImagePath = (): string => {
```
Comment (Bhupesh-mfsi) [inline packages/messenger-web/src/Dm3Widget.tsx:15]: To make the code cleaner please move this function to a separate file **utils/weekPath.ts** and call this here. May be in future we have multiple utility functions and that could make the file look longer.

**Review by Bhupesh-mfsi — CHANGES_REQUESTED** [review]

File: packages/messenger-web/src/Header.tsx [hunk 2]
```diff
@@ -1,54 +1,33 @@
...
-    ) => {
-        e.currentTarget.style.transform = 'scale(1.2)';
-    };
-    const handleMouseOut = (
-        e: React.MouseEvent<HTMLImageElement, MouseEvent>,
-    ) => {
-        e.currentTarget.style.transform = 'scale(1)';
-    };
+    // Use the useMessageTo hook to check if the messageTo parameter is set
+    const [messageTo, isMessageToSet] = useMessageTo();
```
[the remark this answers is not in the document]
Comment (steffenkux) [reply packages/messenger-web/src/Header.tsx:8]: How would I have done it correctly?

[the remark this answers is not in the document]
Comment (Bhupesh-mfsi) [reply packages/messenger-web/src/Header.tsx:8]: Please check the Notification component in this link :

[the remark this answers is not in the document]
Comment (steffenkux) [reply packages/messenger-web/src/Header.tsx]: I added a context and a provider and changed the access. Please check again, if this is best practice or if something should be changed...

[the remark this answers is not in the document]
Comment (Bhupesh-mfsi) [reply packages/messenger-web/src/Header.tsx]: Code looks fine, but the build is getting failed. Please fix that.

[the remark this answers is not in the document]
Comment (Bhupesh-mfsi) [reply packages/messenger-web/src/Header.tsx]: image
Remove this unused property and build will be successfull

File: packages/messenger-web/src/parameter/messageto/MessageToContext.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { createContext, useContext } from 'react';
+import { useMessageTo } from './useMessageTo'; // Import the custom hook to manage the message destination
```
Comment (Bhupesh-mfsi) [inline packages/messenger-web/src/parameter/messageto/MessageToContext.tsx:2]: Please remove this unused property and build will be successfull. This is the reason why build is getting failed

**Pull request merged**